client/components/Login.jsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import axios from 'axios';
+import styles from './Login.module.css';
+
+const Login = () => {
+
+    const [inputEmail, setInputEmail] = useState('');
+    const [inputPassword, setInputPassword] = useState('');
+    const [validEmail, setValidEmail] = useState(false);
+    const [validPassword, setValidPassword] = useState(false);
+
+
+    const onSubmitHandler = (event) => {
+        event.preventDefault();
+        postResponse();
+    };
+
+    const postResponse = async () => {
```

Comment:
when user presses the submit button, it fires this function.
But there's a small problem, we are not validating the inputs here!

If i understand correctly, you are validating using `onBlur` ( which is great! but is it necessary ?)

I think a better approach would be to validate input fields INSIDE the `postResponse` function so that we won't accidentally send invalid input values to the server.

So basically, if ALL input fields are VALID -> send server response. if NOT valid then reveal an error message :)